src/ui/map.js

```diff
@@ -1129,6 +1142,9 @@ class Map extends Camera {
      * Returns a Boolean indicating whether the map's style is fully loaded.
      *
      * @returns {boolean} A Boolean indicating whether the style is fully loaded.
+     *
+     * @example
```

Comment:
Any chance I can add to the scope fo this PR to include improvements to address https://github.com/mapbox/mapbox-gl-js/issues/6708 and https://github.com/mapbox/mapbox-gl-js/issues/6707 as well?